doc/source/examples/bots.rst

```diff
@@ -39,8 +39,8 @@ dependencies.
       - name: automatic merge for Greenkeeper pull requests
         conditions:
           - author=greenkeeper[bot]
-          - status-success=Travis CI - Pull Request
-          - status-sucess=greenkeeper/verify
+          - status-success=continuous-integration/travis-ci/pr
```

Comment:
Please don't change the Travis CI line. Both are valid depending of if you use travis-ci.com or travis-ci.org.

Reply:
Reverted